documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -1018,6 +1018,11 @@ The _semantic type_ describes how the Kafka Connect schema captures the _meaning
 |n/a
 |Contains the string representation of a date range. It always has an exclusive upper-bound.
 
+|`ENUM`
+|`STRING`
+|`io.debezium.data.Enum`
+|Contains the string representation of the PostgreSQL ENUM value
```

Comment:
Also the "allowed" schema parameter schould be mentioned.

Reply:
Fixed